examples/tokenvm/actions/import_asset.go

```diff
@@ -94,7 +94,7 @@ func (i *ImportAsset) executeMint(
 	if err != nil {
 		return utils.ErrBytes(err)
 	}
-	if err := storage.SetAsset(ctx, db, asset, metadata, newSupply, crypto.EmptyPublicKey, true); err != nil {
+	if err := storage.SetAsset(ctx, db, asset, metadata, newSupply, maxSupply, crypto.EmptyPublicKey, true); err != nil {
```

Comment:
If `!exists`, you'll want to set this to `consts.MaxUint64` (or allow for people to configure an amount of assets they are willing to accept from somewhere else in genesis).